docs/gateway/s3.md

```diff
@@ -0,0 +1,51 @@
+# Minio S3 Gateway [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+Minio S3 Gateway adds Minio features like Minio Browser and Caching to AWS S3 buckets.
+
+## Run Minio Gateway for AWS S3
+
+As a pre-requisite to run Minio S3 gateway, you need valid AWS S3 access key and secret key.
+
+### Using Docker
+
+```
+docker run -p 9000:9000 --name minio-s3 \
+ -e "MINIO_ACCESS_KEY=aws_s3_access_key" \
+ -e "MINIO_SECRET_KEY=aws_s3_secret_key" \
+ minio/minio gateway s3
+```
+
+### Using Binary
+
+```
+export MINIO_ACCESS_KEY=aws_s3_access_key
+export MINIO_SECRET_KEY=aws_s3_secret_key
+minio gateway s3
+```
+
+## Minio Caching
+
+Minio caching allows storing content closer to the applications. Frequently accessed objects are stored in the cache. Minio caching with Minio gateway feature allows you to
```

Comment:
suggest 
Minio edge caching allows storing content closer to the applications. Frequently accessed objects are stored in a local disk based cache. Edge caching with Minio gateway feature allows 
 ...